UKHO.PeriodicOutputService/UKHO.BESS.ConfigurationService/Services/ConfigurationService.cs

```diff
@@ -116,31 +118,36 @@ public void ProcessConfigs()
             }
         }
 
-        private IList<BessConfig> DeserializeConfig(string json, string fileName)
+        [ExcludeFromCodeCoverage]
+        private (BessConfig config, bool isValid) DeserializeConfig(string json, string fileName)
         {
-            IList<BessConfig> bessConfig = new List<BessConfig>();
+            BessConfig bessConfig = new();
+            bool isValid = false;
             try
             {
                 var token = JToken.Parse(json);
 
                 if (token.ToString().Contains(UndefinedValue))
                 {
-                    configsWithUndefinedValueCount = configsWithUndefinedValueCount + 1;
-                    logger.LogWarning(EventIds.BessConfigIsInvalid.ToEventId(), "Bess config file : {fileName} contains undefined values or is invalid | _X-Correlation-ID : {CorrelationId}", fileName, CommonHelper.CorrelationID);
+                    filesWithJsonErrorCount = filesWithJsonErrorCount + 1;
+                    logger.LogWarning(EventIds.BessConfigValueNotDefined.ToEventId(), "Bess config file : {fileName} contains undefined values. | _X-Correlation-ID : {CorrelationId}", fileName, CommonHelper.CorrelationID);
                 }
                 else
                 {
-                    bessConfig = JsonConvert.DeserializeObject<List<BessConfig>>(json)!;
+                    bessConfig = JsonConvert.DeserializeObject<BessConfig>(json)!;
+                    isValid = true;
                 }
-                return bessConfig;
+                return new(bessConfig, isValid);
             }
             catch (Exception ex)
             {
-                logger.LogError(EventIds.BessConfigParsingError.ToEventId(), "Error occurred while parsing Bess config file : {fileName} | Exception Message : {Message} | StackTrace : {StackTrace} | _X-Correlation-ID : {CorrelationId}", fileName, ex.Message, ex.StackTrace, CommonHelper.CorrelationID);
-                return bessConfig;
+                filesWithJsonErrorCount = filesWithJsonErrorCount + 1;
+                logger.LogError(EventIds.BessConfigParsingError.ToEventId(), "Error occurred while parsing Bess config file : {fileName}. It might have  missing or extra commas, missing brackets, or other syntax errors.| Exception Message : {Message} | StackTrace : {StackTrace} | _X-Correlation-ID : {CorrelationId}", fileName, ex.Message, ex.StackTrace, CommonHelper.CorrelationID);
```

Comment:
Need to be changes in UTs as well.
```suggestion
                logger.LogError(EventIds.BessConfigParsingError.ToEventId(), "Error occurred while parsing Bess config file : {fileName}. It might have missing or extra commas, missing brackets, or other syntax errors.| Exception Message : {Message} | StackTrace : {StackTrace} | _X-Correlation-ID : {CorrelationId}", fileName, ex.Message, ex.StackTrace, CommonHelper.CorrelationID);
```